<?xml version="1.0"?>
<launch>
 <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node"/>
 <node pkg="turtle2pose" type="pose_server_node" name="pose_server_node"/>
 <node pkg="turtle_control_action" type="action_server_node" name="action_server_node" output="screen"/>
 <node pkg="turtle_control_action" type="action_client_node" name="action_client_node" output="screen"/>
 <rosparam file="$(find turtle_control_action)/param/path.yaml" command="load"/><!--别忘记装载param-->
</launch>
